bib/internal/utils/utils.go

```diff
@@ -0,0 +1,11 @@
+package utils
+
+import "os/exec"
+
+// IsMountpoint checks if the target path is a mount point
+func IsMountpoint(path string) bool {
+	c := exec.Command("mountpoint", path)
+	c.Stderr = nil
```

Comment:
(nitpick) is this necessary? Iirc that is the default [0] - or am I missing something?


[0] https://github.com/golang/go/blob/master/src/os/exec/exec.go#L376

Reply:
You're right; fixed